internal/v2/controller/http/common.go

```diff
@@ -0,0 +1,51 @@
+package http
+
+import (
+	"net/http"
+
+	sdkCommon "github.com/edgexfoundry/device-sdk-go/internal/common"
+	"github.com/edgexfoundry/device-sdk-go/internal/container"
+	"github.com/edgexfoundry/device-sdk-go/internal/telemetry"
+	"github.com/edgexfoundry/go-mod-bootstrap/di"
+	contractsV2 "github.com/edgexfoundry/go-mod-core-contracts/v2"
+	"github.com/edgexfoundry/go-mod-core-contracts/v2/dtos/common"
+)
+
+// Ping handles the request to /ping endpoint. Is used to test if the service is working
+// It returns a response as specified by the V2 API swagger in openapi/v2
+func (c *V2HttpController) Ping(writer http.ResponseWriter, request *http.Request, _ *di.Container) {
+	response := common.NewPingResponse()
+	c.sendResponse(writer, request, contractsV2.ApiPingRoute, response, http.StatusOK)
+}
+
+// Version handles the request to /version endpoint. Is used to request the service's versions
+// It returns a response as specified by the V2 API swagger in openapi/v2
+func (c *V2HttpController) Version(writer http.ResponseWriter, request *http.Request, _ *di.Container) {
+	response := common.NewVersionResponse(sdkCommon.V2ServiceVersion)
```

Comment:
Needs to use common.NewVersionSdkResponse() and include the SDKversion

Reply:
updated.

Reply:
noticed that although in [this discussion](https://github.com/edgexfoundry/device-sdk-go/pull/549#discussion_r460215570) we decided to leave it fully until v2, but seems passing `SDKVersion` during build time won't break any compatibility so I'll just make the change now.

Reply:
Correct.